src/Commands/CommentAddPullRequestCommand.php

```diff
@@ -0,0 +1,60 @@
+<?php
+/**
+ * Terminus Plugin that contain a collection of commands useful during
+ * the build step on a [Pantheon](https://www.pantheon.io) site that uses
+ * a GitHub PR workflow.
+ *
+ * See README.md for usage information.
+ */
+
+namespace Pantheon\TerminusBuildTools\Commands;
+
+use Pantheon\Terminus\Exceptions\TerminusException;
+
+/**
+ * Pull Request Comment Command
+ */
+class CommentAddPullRequestCommand extends BuildToolsBase
+{
+
+    /**
+     * Add a comment to a specified PR on the repository.
+     *
+     * @authorize
+     *
+     * @command build:comment:add:pr
+     */
+    public function commentAddPullRequest(
+        $options = [
+            'pr_id' => '',
+            'message' => '',
+            'site_url' => ''
+        ])
+    {
+        // Get current repository and commit
+        $remoteUrlFromGit = exec('git config --get remote.origin.url');
+        $prId = $options['pr_id'];
+        if (empty($prId)) {
+            throw new TerminusException( '--pr_id=<id> is required.' );
+        }
+
+        // Create a Git repository service provider appropriate to the URL
+        $this->inferGitProviderFromUrl($remoteUrlFromGit);
+
+        // Ensure that credentials for the Git provider are available
+        $this->providerManager()->validateCredentials();
+
+        // Compile message
+        if (!empty($options['site_url'])) {
```

Comment:
I think we should prioritize a user-passed message first, then fall back to constructing a message from `site_url`.

Reply:
I copied this from `CommentAddCommitCommand`. Should we change that as well for consistency, or should we make this change in a follow-up PR?

Reply:
A follow-up PR sounds good to reduce bloat here.